features/admin.extensions.v1/application-templates/shared/components/m2m-custom-configuration.tsx

```diff
@@ -250,36 +254,38 @@ export const M2MCustomConfiguration: FC<M2MCustomConfigurationPropsInterface> =
                                         className="mt-2"
                                     />
                                 </Form.Field>
-                                <Form.Field>
-                                    <label>
-                                        {
-                                            t("extensions:develop.applications.quickstart.mobileApp" +
-                                                ".configurations.scope.label")
-                                        }
-                                        <Hint className="mt-0 mb-0" popup>
-                                            <Trans
-                                                i18nKey={
-                                                    "extensions:develop.applications.quickstart" +
-                                                    ".m2m.configurations.scopes"
-                                                }
-                                            >
-                                                The list of authorized scopes.
-                                                Include required scopes by authorizing APIs through the
-                                                <a
-                                                    className="link pointing"
-                                                    onClick={ onAPIAuthorizationTabClick }
+                                { isScopeAvailable(copyScopesValue) && (
```

Comment:
can't we simply use `!!copyScopesValue` here?

```suggestion
                                { !!copyScopesValue && (
```

Reply:
Also, have we considered showing a warning/info message when scopes field is not available?

Reply:
Fixed with: https://github.com/wso2/identity-apps/pull/6471/commits/2dd27a320bfa6287ea8a9fda305d7b85649a74f2

Reply:
`have we considered showing a warning/info message when scopes field is not available?` Previously we were displaying an info message with the empty field, but then it was decided to remove the scope field when API's are not authorized.